components/eam/src/control/cam_comp.F90

```diff
@@ -137,11 +138,11 @@ subroutine cam_init( cam_out, cam_in, mpicom_atm, &
    logical :: log_print        ! Flag to print out log information or not
    character(len=cs) :: filein ! Input namelist filename
 #if defined(CLDERA_PROFILING)
-   character(len=max_str_len), allocatable :: cldera_field_names(:)
```

Comment:
Is this not needed anymore?

Reply:
Yeah, it wasn't needed after a previous PR, but was left in there. Unfortunately, given the *vast* amount of warnings coming from eam/homme, we never noticed this, so the compiler warning got undetected.